open-data-fabric.md

```diff
@@ -1854,6 +1856,19 @@ Reader for Apache Parquet format.
 [^](#reference-information)
 
 
+<a name="requestheader-schema"></a>
+##### RequestHeader
```

Comment:
Looks like someone (me) forgot to run `make` the last time I modified the schemas :(